run_tests.py

```
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
```

Ruff (0.13.2)

Shebang is present but file is not executable

(EXE001)